client-cli/src/command.rs

```diff
@@ -53,6 +53,8 @@ ENVIRONMENT VARIABLES:
     CRYPTO_CHAIN_ID                 Chain ID of Crypto.com Chain
     CRYPTO_CLIENT_STORAGE           Storage directory (Default: `.storage`)
     CRYPTO_CLIENT_TENDERMINT        Websocket endpoint for tendermint (Default: `ws://localhost:26657/websocket`)
+    PASSPHRASE                      Passphrase to use when needed
+    AUTH_TOKEN                      Auth token to use when needed
```

Comment:
users will not know difference between passphrase and auth_token
add some explanation for above

Reply:
also names
CRYPTO_PASSPHRASE
CRYPTO_AUTH_TOKEN